File: LinkedList/PHP/DoublyLinkedListNode.php
<?php
/**
 * Created by PhpStorm.
 * User: saeed
 * Date: 24/11/16
 * Time: 05:18 AM
 */

namespace LinkedList;


class DoublyLinkedListNode
{
    public $value;
    public $next;
    public $previous;

    public function __construct($value)
    {
        $this->previous = null;
        $this->next = null;
        $this->value = $value;
    }
}

File: LinkedList/C#/LinkedList/LinkedList/DoublyLinkedListNode.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LinkedList
{
    class DoublyLinkedListNode<T>
    {
        public T Value { set; get; }
        public DoublyLinkedListNode<T> Previous { set; get; }
        public DoublyLinkedListNode<T> Next { set; get; }
        public DoublyLinkedListNode()
        {
            this.Previous = null;
            this.Next = null;
        }
        public DoublyLinkedListNode(T value)
        {
            this.Value = value;
            this.Previous = null;
            this.Next = null;
        }
    }
}
